Editor/ExportUtility.cs

```diff
@@ -1351,6 +1357,14 @@ public static void ExportDynamicObjects(List<DynamicObject> dynamicObjects, bool
                 //if (dynamicObject == null) { return false; }
                 if (dynamicObject == null) { continue; }
 
+                // handle the dynamic objects which are TextMeshPros
+                if (dynamicObject.gameObject.GetComponent<TextMeshPro>() != null)
+                {
+                    GameObject go = ExportQuad(dynamicObject.gameObject, temporaryDynamicMeshes, ExportQuadType.TMPro, true);
+                    dynamicObjects.Add(go.GetComponent<DynamicObject>());
```

Comment:
This will be added to end of list, so will be handled later. Feels strange - is it bad practice?

Reply:
Hmm, throws an error 

[as shown here](https://stackoverflow.com/questions/604831/collection-was-modified-enumeration-operation-may-not-execute)

Reply:
Following suggested changes on this link worked